iphone/Classes/PlatformModule.m

```diff
@@ -266,31 +277,33 @@ - (NSNumber *)availableMemory
   return [NSNumber numberWithDouble:((vm_page_size * vmStats.free_count) / 1024.0) / 1024.0];
 }
 
-- (NSNumber *)openURL:(NSArray *)args
+- (BOOL)openURL:(NSString *)newUrlString
```

Comment:
And another one: Can we unbox the first arg automatically now? What happens with > 1 args?